lib/rucio/daemons/replicarecoverer/suspicious_replica_recoverer.py

```diff
@@ -25,11 +25,13 @@
 import re
 import threading
 import time
+from collections.abc import Callable
```

Comment:
`Callable` is only used in the context of type-checking here, so it can be added to the `if TYPE_CHECKING` block